fastlane/Fastfile

```diff
@@ -193,3 +193,16 @@ lane :test do
     notify_testers: false
   )
 end
+
+lane :test_fetch_version_number do
+
+  UI.message("\n\n\n=====================================\n Fetch latest version: \n=====================================")
+
+  version = appcenter_fetch_version_number(
+    api_token: ENV["TEST_APPCENTER_API_TOKEN"],
+    owner_name: ENV["TEST_APPCENTER_OWNER_NAME"],
+    app_name: "MyApplication"
+  )
+
+  UI.message("#{version}")
```

Comment:
Rubocop is complaining here. This is pretty irrelevant, to be honest, but for your information here's what I get when running `rake`:
```
fastlane/Fastfile:198:1: C: Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body beginning.
fastlane/Fastfile:207:14: C: Style/RedundantInterpolation: Prefer to_s over string interpolation.
  UI.message("#{version}")
             ^^^^^^^^^^^^

6 files inspected, 2 offenses detected
```